### ISSUE TITLE
ORNL DAAC identify tool

### ISSUE THREAD

**Issue by mdietze** [PecanProject/pecan#830]
The DAAC developed a tools that pulls out a bunch of their data layers for a single point
http://webmap.ornl.gov/qaqc/identify/

They developed this to help with their QAQC but I think this could help with both the site definition and the site info -- this is some of the exact stuff that I was hoping we could display on the google pin #353 and is also data we could ingest into write configs as well.

**Comment by serbinsh** [comment]
Yes, this does look very handy and would enable quick OTF viewing of site stats when clicked on the map! However, it did take awhile for it to report several characteristics at once in the table below.  So the speed of reporting might be an issue and could cause some latency issues....just something we should keep on eye on.
